apps/bluetooth/bluetooth-service.js

```diff
@@ -0,0 +1,252 @@
+'use strict'
+
+var Service = require('@yodaos/application').Service
+var logger = require('logger')('bluetooth-service')
+var bluetooth = require('@yoda/bluetooth')
+var protocol = bluetooth.protocol
+var util = require('util')
+var _ = require('@yoda/util')._
+// var system = require('@yoda/system')
+// var httpUtil = require('./http-util.js')
+var wifi = require('@yoda/wifi')
```

Comment:
The package `@yoda/wifi` has been deprecated, please using the alternative package `@yoda/network`.

Reply:
Replaced by `@yoda/network`